### PR TITLE
Security section for Umbraco-Cloud

### PR DESCRIPTION
**Pull request by pijemcolu**
Describes:

HTTPS & Certificates
TLS protocol support & TLS ciphers
Firewall & Access restriction features

### CODE REVIEW

File: Umbraco-Cloud/Security/index.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+All Umbraco Cloud websites use HTTPS by default. Both the default {projectName}.s1.umbraco.io and custom domains are protected by periodically renewed certificates from Let's Encrypt. This service is offered as part of Umbraco Cloud by [Umbraco LATCH](/Umbraco-Cloud/Set-Up/Umbraco-Latch/index.md)
+
+### Custom Certificates
+
+Custom certificates can be used with all custom domains. Please refer to [Managing Custom Certificates documentation](Umbraco-Cloud/Set-Up/Manage-Hostnames/Security-Certificates/index.md)
+
+### TLS support
+
+As of April 2020, we've deprecated support for TLS1.0 & TLS 1.1. TLS1.2 is the only supported TLS protocol going forward.
```
Comment (sitereactor) [inline Umbraco-Cloud/Security/index.md]: Should this say "minimum supported version" ?

File: Umbraco-Cloud/Security/index.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+- TLS_DHE_RSA_WITH_AES_256_CBC_SHA
+- TLS_DHE_RSA_WITH_AES_128_CBC_SHA
+
+### TLS1.2 by default in external services
+
+In order to integrate older external applications accessing Umbraco Cloud Websites you might have to modify the TLS support in the .Net application.
+
+For ASP.NET applications, inspect the `<system.web><httpRuntime targetFramework>` element of web.config to find the version of the .NET Framework your application is using. .NET applications on .NET 4.7+ are using the OS specified TLS protocols. In Windows 8 & 10, Windows Server 2012 & 2016 that's 1.2+ by default, therefore no actions necessary. .NET applications lower then 4.7 require updates to ensure they can communicate using TLS1.2 by default.
+
+More information specifically from MS about .Net applications and TLS support can be found [in MS docs](https://docs.microsoft.com/en-us/dotnet/framework/network-programming/tls#audit-your-code-and-make-code-changes). For other application frameworks/languages we encourage to lookup their respective documentations.
```
Comment (sitereactor) [inline Umbraco-Cloud/Security/index.md]: write Microsoft rather then MS

**Comment by sitereactor** [comment]
Two suggestions....other then that it looks good to me.

File: Umbraco-Cloud/Security/index.md (hunk 1 above)
Comment (sitereactor) [reply Umbraco-Cloud/Security/index.md]: Should we explicitly state that we add support for TLS 1.2 for the older .net framework versions where TLS 1.0 was the default and TLS 1.2 wasn't added by default? I believe this is the case for sites on Cloud running Umbraco 7.x and .net framework 4.5.2.

File: Umbraco-Cloud/Security/index.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
+# Security on Umbraco Cloud
+
+## HTTPS & Certificates
+
+All Umbraco Cloud websites use HTTPS by default. Both the default {projectName}.s1.umbraco.io and custom domains are protected by periodically renewed certificates from Let's Encrypt. This service is offered as part of Umbraco Cloud by [Umbraco LATCH](/Umbraco-Cloud/Set-Up/Umbraco-Latch/index.md)
+
+### Custom Certificates
+
+Custom certificates can be used with all custom domains. Please refer to [Managing Custom Certificates documentation](Umbraco-Cloud/Set-Up/Manage-Hostnames/Security-Certificates/index.md)
```
Comment (sofietoft) [inline Umbraco-Cloud/Security/index.md]:
```suggestion
Custom certificates can be used with all custom domains. Please refer to our [Managing Custom Certificates documentation](Umbraco-Cloud/Set-Up/Manage-Hostnames/Security-Certificates)
```

File: Umbraco-Cloud/Security/index.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
+# Security on Umbraco Cloud
+
+## HTTPS & Certificates
+
+All Umbraco Cloud websites use HTTPS by default. Both the default {projectName}.s1.umbraco.io and custom domains are protected by periodically renewed certificates from Let's Encrypt. This service is offered as part of Umbraco Cloud by [Umbraco LATCH](/Umbraco-Cloud/Set-Up/Umbraco-Latch/index.md)
```
Comment (sofietoft) [inline Umbraco-Cloud/Security/index.md]:
```suggestion
All Umbraco Cloud websites use HTTPS by default. Both the default {projectName}.s1.umbraco.io and custom domains are protected by periodically renewed certificates from Let's Encrypt. This service is offered as part of Umbraco Cloud by [Umbraco LATCH](/Umbraco-Cloud/Set-Up/Umbraco-Latch).
```

File: Umbraco-Cloud/Security/index.md (hunk 1 above)
Comment (sofietoft) [inline Umbraco-Cloud/Security/index.md]:
```suggestion
As of April 2020, we've deprecated support for TLS 1.0 & TLS 1.1. TLS 1.2 is the only supported TLS protocol going forward.
```

File: Umbraco-Cloud/Security/index.md [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+- TLS_ECDHE_RSA_WITH_AES_256_CBC_SHA384
+- TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA256
+- TLS_ECDHE_RSA_WITH_AES_256_CBC_SHA
+- TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA
+- TLS_DHE_RSA_WITH_AES_256_GCM_SHA384
+- TLS_DHE_RSA_WITH_AES_128_GCM_SHA256
+- TLS_DHE_RSA_WITH_AES_256_CBC_SHA
+- TLS_DHE_RSA_WITH_AES_128_CBC_SHA
+
+### TLS1.2 by default in external services
```
Comment (sofietoft) [inline Umbraco-Cloud/Security/index.md]:
```suggestion
### TLS 1.2 by default in external services
```

File: Umbraco-Cloud/Security/index.md [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+
+In order to integrate older external applications accessing Umbraco Cloud Websites you might have to modify the TLS support in the .Net application.
```
Comment (sofietoft) [inline Umbraco-Cloud/Security/index.md]:
```suggestion
In order to integrate older external applications by accessing Umbraco Cloud Websites you might have to modify the TLS support in the .NET application.
```

File: Umbraco-Cloud/Security/index.md [hunk 7]
```diff
@@ -0,0 +1,46 @@
...
+
+For ASP.NET applications, inspect the `<system.web><httpRuntime targetFramework>` element of web.config to find the version of the .NET Framework your application is using. .NET applications on .NET 4.7+ are using the OS specified TLS protocols. In Windows 8 & 10, Windows Server 2012 & 2016 that's 1.2+ by default, therefore no actions necessary. .NET applications lower then 4.7 require updates to ensure they can communicate using TLS1.2 by default.
```
Comment (sofietoft) [inline Umbraco-Cloud/Security/index.md]:
```suggestion
For ASP.NET applications, inspect the `<system.web><httpRuntime targetFramework>` element of web.config to find the version of the .NET Framework your application is using. .NET applications on .NET 4.7+ are using the OS specified TLS protocols. In Windows 8 & 10, Windows Server 2012 & 2016 TLS 1.2+ is used by default, therefore no actions necessary. .NET applications lower then 4.7 require updates to ensure they can communicate using TLS 1.2 by default.
```

File: Umbraco-Cloud/Security/index.md [hunk 8]
```diff
@@ -0,0 +1,46 @@
...
+
+More information specifically from MS about .Net applications and TLS support can be found [in MS docs](https://docs.microsoft.com/en-us/dotnet/framework/network-programming/tls#audit-your-code-and-make-code-changes). For other application frameworks/languages we encourage to lookup their respective documentations.
+
+### HTTP
+
+HTTP protocol is supported but not used by default on Umbraco Cloud Websites. If you'd like to keep using HTTP, which we strongly discourage, you'll need to remove a web.config transform as specified in [Umbraco LATCH documentation](/Umbraco-Cloud/Set-Up/Umbraco-Latch/index.md)
```
Comment (sofietoft) [inline Umbraco-Cloud/Security/index.md]:
```suggestion
HTTP protocol is supported but not used by default on Umbraco Cloud Websites. If you'd like to keep using HTTP, which we strongly discourage, you'll need to remove a web.config transform as specified in [Umbraco LATCH documentation](/Umbraco-Cloud/Set-Up/Umbraco-Latch)
```

**Comment by sofietoft** [comment]
Also, you don't think we should add the content from here as well: https://our.umbraco.com/documentation/Umbraco-Cloud/Frequently-Asked-Questions/#security-and-encryption ? 🤔 
I think it would be great to have all info about security added to a single article.
I'd be happy to do that after merge, if you think it's a good idea 💪

File: Umbraco-Cloud/Security/index.md (hunk 1 above)
Comment (pijemcolu) [reply Umbraco-Cloud/Security/index.md]: @sitereactor we only support TLS1.2, we don't support 1.3, that means 1.2 is the only supported version. We could remove "only".

I've now specifically mentioned changing the default TLS version for version 7 websites.

File: Umbraco-Cloud/Security/index.md (hunk 2 above)
Comment (pijemcolu) [reply Umbraco-Cloud/Security/index.md]: Did so.

File: Umbraco-Cloud/Security/index.md (hunk 6 above)
Comment (pijemcolu) [reply Umbraco-Cloud/Security/index.md]: It's the external applications which are accessing Umbraco Cloud Websites. Adding "by" changes the meaning. Should we change this into something else then?

**Comment by pijemcolu** [comment]
I'd agree we should add that in. I'm unsure ion what format and whether it's a good idea to put all those quetions in there. That's why I didn't touch it.

File: Umbraco-Cloud/Security/index.md (hunk 1 above)
Comment (sitereactor) [reply Umbraco-Cloud/Security/index.md]: Sure I get that. I just thought that "minimum" would be better, so we don't have something that is potentially wrong for certain parts of the platform when the hosting changes or when we have a better DNS/CDN provider in front. But fine to leave this for now.

File: Umbraco-Cloud/Security/index.md (hunk 6 above)
Comment (sofietoft) [reply Umbraco-Cloud/Security/index.md]: Ah, I see 🤔
Perheps we could say `to` instead?
```suggestion
In order to integrate older external applications to access Umbraco Cloud Websites you might have to modify the TLS support in the .Net application.
```

That keeps the origin meaning, right?

**Comment by sofietoft** [comment]
I'll look into merging the two section together, and try to keep some of the info on the FAQ page as well 💪

**Pull request merged**